### DISCUSSION TITLE
whisper-cpp is an old version - v1.5.4 rather than v1.7.1

### DISCUSSION THREAD

**Discussion by jamescridland** [Homebrew/discussions#5658] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: 84c31175f11860129a9aaed40a13c549625e2db1
Last commit: 7 days ago
Core tap JSON: 08 Oct 18:56 UTC
Core cask tap JSON: 08 Oct 18:56 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.5 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.5/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 16.0.0 build 1600
Git: 2.47.0 => /opt/homebrew/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.0.1-arm64
CLT: 16.0.0.0.1.1724870825
Xcode: 16.0
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Unexpected dylibs:
  /usr/local/lib/Spyder.dylib
  /usr/local/lib/Spyder3.dylib
  /usr/local/lib/Spyder5.dylib
  /usr/local/lib/dccmtr.dylib
  /usr/local/lib/dcxmtr.dylib
  /usr/local/lib/libCIEHandler.dylib
  /usr/local/lib/libCalibrationHandler.dylib
  /usr/local/lib/libDeviceManager.dylib
  /usr/local/lib/libErrorHandler.dylib
  /usr/local/lib/libICMManager.dylib
  /usr/local/lib/libMCCAppManager.dylib
  /usr/local/lib/libMCCEngineWrapper.dylib
  /usr/local/lib/libMonitorDDCCISDK.dylib
  /usr/local/lib/libProtocolEngine.dylib
  /usr/local/lib/libmkusb.dylib
  /usr/local/lib/libusb-1.0.0.dylib
```

### Description of issue

I love using brew, it's an excellent piece of software and I'm ever grateful for the people that maintain each cask so that my Mac works brilliantly.

Now - `whisper-cpp` is currently v 1.5.4 in homebrew...

```
==> whisper-cpp: stable 1.5.4 (bottled), HEAD
Port of OpenAI's Whisper model in C/C++
https://github.com/ggerganov/whisper.cpp
Installed
/opt/homebrew/Cellar/whisper-cpp/1.5.4 (9 files, 2.0MB) *
  Poured from bottle using the formulae.brew.sh API on 2024-05-27 at 16:13:22
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/w/whisper-cpp.rb
License: MIT
```

The new version of whisper-cpp [on its own Github](https://github.com/ggerganov/whisper.cpp/tree/master) is v1.7.1 - which is a big update, since it now supports the much-faster and much-better large-turbo model. I'm quite keen to use it, but I also am not that confident in compiling from source - and certainly not that confident in submitting a pull request to help update it even if I was.

I don't know if this is generally done here, but I'd love to see the version updated within homebrew - and if anyone's able to do that, I'll be your best friend.

### Relevant casks

whisper-cpp

**Comment by p-linnane** [comment, marked as answer]
Looks like this wasn't added to our autobump list. I've opened a PR to update `whisper-cpp` and add it to our autobump list: https://github.com/Homebrew/homebrew-core/pull/193264

**Reply by jamescridland** [reply]
Thanks, new best friend!

**Reply by jamescridland** [reply]
@p-linnane Just wondering when we might see the update appear? Homebrew is still reporting that it's v1.5.4.

It looks as though there is a maintainer who has made the changed, but there's an open conversation about it.

**Reply by SMillerDev** [reply]
It didn't work, so the pull request was closed. It would be best to make a pull request that updates it and fixes the build yourself.
